gradle/versions.gradle

```diff
@@ -54,9 +54,10 @@ dependencyManagement {
 
     dependency 'org.mock-server:mockserver-netty:5.5.1'
 
-    dependency 'org.web3j:abi:4.3.0'
-    dependency 'org.web3j:core:4.3.0'
-    dependency 'org.web3j:crypto:4.3.0'
-    dependency 'org.web3j:eea:4.3.0'
+    dependency 'org.web3j:abi:4.4.0'
+    dependency 'org.web3j:core:4.4.0'
+    dependency 'org.web3j:crypto:4.4.0'
+    dependency 'org.web3j:eea:4.4.0'
```

Comment:
You can drop `eea` - you should be able to drop `core` too unless there are modules that you would like only the `core` functionality in i.e. you don't need the whole `pantheon` module for public transactions. 

Also, the latest version is 4.5.0 - the main change here is the addition of `Base64String` to solve character encoding problems. The means you can drop some code in `EeaTransaction` too :)

Reply:
upgraded to 4.5.0 - removed unnecessary web3j references (nice) - but needed to keep some of them